src/grpc/server.rs

```diff
@@ -37,16 +35,22 @@ impl ConsumerService for TaskbrokerServer {
                     inflight.activation.received_at.unwrap().seconds,
                     inflight.activation.received_at.unwrap().nanos as u32,
                 ) {
+                    let mut received_to_gettask_latency = Utc::now()
+                        .signed_duration_since(sentry_ts)
+                        .num_milliseconds()
+                        as f64;
+                    if let Some(delay_until) = inflight.delay_until {
+                        received_to_gettask_latency -= delay_until
+                            .signed_duration_since(sentry_ts)
+                            .num_milliseconds()
+                            as f64;
```

Comment:
This feels like we could have a method on `InflightActivation` that does this logic. Having a method would help reduce duplication.